Automattic/jetpack
# Licensing: My Jetpack: Add and update entry points into licensing activation UI in My Jetpack

#### Changes proposed in this Pull Request:

* Now that the licensing activation UI can also be loaded in My Jetpack, we can being transition entry points to point to My Jetpack. This PR does that, while also checking for the licensing activation UI feature flag
* As part of this work, this PR updates the links in the primary My Jetpack page to use the button component in `@automattic/jetpack-components` which results in a thicker underline on hover. 

#### Jetpack product discussion



NA

#### Does this pull request change what data or activity we track or use?



Yes. There is a new Tracks event that is added here for users that click the back button within the licensing activation UI. `jetpack_myjetpack_license_activation_back_link_click`

Though, I do not think this is significant enough to need to update documentation. 

#### Testing instructions:






* Checkout PR
* `jetpack build plugins/jetpack`
* Go to Jetpack > Dashboard > My Plan
* Click "Activate a Product" button
* Ensure that you stay in the Jetpack plugin UI the URL does not have My Jetpack
* Enable licensing activation UI feature flag with `define( 'JETPACK_ENABLE_MY_JETPACK_LICENSE', true );`
* Go to Jetpack > Dashboard > My Plan
* Click "Activate a Product" button
* Ensure that you end up in the licensing activation UI in My Jetpack (url should have my-jetpack in it
* Click "Back" button
* Ensure that you end back up at the My Plan page
* Go to "My Jetpack"
* Ensure that you are user connected
* Ensure that "Activate a license" link shows in bottom left
* Click that link
* Ensure that you end up in licensing activation UI within my Jetpack
* Click "Back" button
* Ensure that you end back up on My Jetpack

See [video](https://user-images.githubusercontent.com/1126811/165997924-c9b8ddc0-313a-41fc-bf78-e784fabbe4b7.mov)

https://user-images.githubusercontent.com/1126811/165997924-c9b8ddc0-313a-41fc-bf78-e784fabbe4b7.mov
